cyclomatic_complexity/init.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
 WORK_DIR=${1}
-INITIAL_PROJECT_LOCATION=${2}
-SHARED_API_DIR=${2}/api.pmccabe_collector.restapi.org
+export INITIAL_PROJECT_LOCATION=${2}
 MAIN_IMAGE_ENV_SHARED_LOCATION=${3}
+SHARED_API_DIR=${4}
 
-echo -e "#!/bash\n\nexport WORK_DIR=${WORK_DIR} \nexport MAIN_IMAGE_ENV_SHARED_LOCATION=${MAIN_IMAGE_ENV_SHARED_LOCATION} \nexport SHARED_DIR=${SHARED_DIR} \nexport SHARED_API_DIR=${SHARED_API_DIR} \nexport INITIAL_PROJECT_LOCATION=${INITIAL_PROJECT_LOCATION} \nexport RRD_ROOT=${SHARED_API_DIR}" > ${MAIN_IMAGE_ENV_SHARED_LOCATION}/setenv.sh
+MAIN_SERVICE_NAME=api.pmccabe_collector.restapi.org
+echo -e "#!/bash\n\nexport WORK_DIR=${WORK_DIR} \nexport MAIN_IMAGE_ENV_SHARED_LOCATION=${MAIN_IMAGE_ENV_SHARED_LOCATION} \nexport SHARED_DIR=${SHARED_DIR} \nexport SHARED_API_DIR=${SHARED_API_DIR} \nexport INITIAL_PROJECT_LOCATION=${INITIAL_PROJECT_LOCATION} \nexport MAIN_SERVICE_NAME=${MAIN_SERVICE_NAME}" > ${MAIN_IMAGE_ENV_SHARED_LOCATION}/setenv.sh
```

Comment:
reconsider using export ENV variables instead of producing such `setenv.sh` file

Reply:
The issue #30 created